storage/boltdb/logger.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package boltdb
+
+import (
+	"go.uber.org/zap"
+
+	"storj.io/storj/storage"
+	"storj.io/storj/storage/storelogger"
+)
+
+// NewClient instantiates a new BoltDB client given db file path, and a bucket name
+func NewClient(log *zap.Logger, path, bucket string) (storage.KeyValueStore, error) {
+	client, err := New(path, bucket)
+	db := storelogger.New(log, client)
+	return db, err
+}
+
+// NewDebug instantiates a new BoltDB client given db file path, and a bucket name
+func NewDebug(path, bucket string) (storage.KeyValueStore, error) {
+	lg, _ := zap.NewDevelopment()
```

Comment:
if you just use the global zap logger (zap.L()), then it is already configured by our configuration system to allow you to set Development vs Production mode. i think i'd prefer to avoid instantiating more logger configurations in other code places. see pkg/process/exec_conf.go, search for `defer zap.ReplaceGlobals`